src/vcpkg/tools.cpp

```diff
@@ -587,46 +605,26 @@ gsutil version: 4.58
                 // location.
                 if (tool == Tools::CMAKE)
                 {
-                    if (get_environment_variable("VCPKG_FORCE_SYSTEM_BINARIES").has_value())
-                    {
-                        return {"cmake", "0"};
-                    }
-                    return get_path(paths, CMakeProvider());
+                    get_system_tool_path(paths, "cmake");
```

Comment:
This looks like it would cause a lot of binary cache key misses because the CMake version is included in the vcpkg_abi_info.txt. See also: https://github.com/microsoft/vcpkg/issues/16615

Reply:
Sorry I think I got confused by the name of the new `get_system_tool_path` function introduced in this PR. From the name of the function I expected it to unconditionally use the system version of the tool rather than a copy vcpkg downloads. But it is a wrapper around `get_path`.